vendor/github.com/containers/image/v4/copy/copy.go

```diff
@@ -0,0 +1,975 @@
+package copy
```

Comment:
All of these additions should, ideally, be moves that just add the `/v4/` path prefix.

It may not be possible or worth it, but we should in that case understand why.

Reply:
It has already been moved to `/v4/`, unless I am missing something here.

Reply:
Yes, it’s fine now; IIRC it wasn’t previously and this was adding a new v4 copy while leaving pre-v4 packages around in `vendor`.